src/Microsoft.ML.Data/Transforms/ConversionsExtensionsCatalog.cs

```diff
@@ -141,7 +130,7 @@ public static KeyToValueMappingEstimator MapKeyToValue(this TransformsCatalog.Co
             this TransformsCatalog.ConversionTransforms catalog,
             IEnumerable<TInputType> keys,
             IEnumerable<TOutputType> values,
-            params (string source, string name)[] columns)
+            params (string inputColumn, string outputColumn)[] columns)
```

Comment:
>inputColumn, string outputColumn [](start = 27, length = 32)

Remember, `name`, `source`, not `source`, `name`. And definitely *not* `input`, `output`. :)

Reply:
Also I think for conistency sake we want the column mapping to be at the head, even when we have to forgo the convenience of the `params`.

---
In reply to: [248079402](https://github.com/dotnet/machinelearning/pull/2118#discussion_r248079402) [](ancestors = 248079402)

Reply:
I will do this on the next PR, because the tuple is passed directly from the estimator -> transfomer-> OneToOnetransformer base class. 

I agree that it is confusing to review, because i did partial work with renaming the tuple items. 

---
In reply to: [248079667](https://github.com/dotnet/machinelearning/pull/2118#discussion_r248079667) [](ancestors = 248079667,248079402)